docs/content/guides/6.multistore/2.tooling-and-concepts/3.development/3.managing-the-stores.md

```diff
@@ -7,11 +7,279 @@ navigation:
 
 # Development - Managing the stores
 
+This guide covers the core operations for managing stores in Alokai multistore projects through the Alokai CLI.
+
+**What You'll Learn**
+
+::list{type="success"}
+- How to create new stores using the CLI
+- Understanding and modifying store configurations
+- Moving stores within your project hierarchy
+- Safely removing stores and handling cleanup
+::
+
+**Why This Matters**
+
+When working with multiple brands and regions, keeping clean and maintainable repository is one of the most important characteristics that ensure you can keep releasing fast and with a high confidence. The Alokai CLI provides the tools you need to create, configure, and manage stores while keeping your codebase clean and structured.
+
+Let's explore how to manage your stores effectively.
+
+## Adding a New Store
+
+The `store add` command guides you through creating a new store and sets up all necessary configurations:
+
+```bash
+yarn store add
+```
+
+### Store Creation Flow
+
+The CLI will ask several questions to configure your store:
+
+1. **Store ID**
+   - Unique identifier for your store
+   - Used in `alokai.config.json`, `turbo.json`, and store directory
+   - Example: `my-brand-us`
+
+2. **Parent Store**
+   - Choose if this store inherits from existing store
+   - Or choose root to inherit from base apps
+
+3. **Store Type**
+   - Choose if store should be deployable or not. Template stores are not deployable.
+
+4. **Console Project** (for deployable stores only)
+   - The project name in Alokai Console
+   - If you're not sure, just skip it for now
+   - Can be updated later in `alokai.config.json`
+
+5. **Cloud Region** (for deployable stores only)
+   - Deployment region for your store
+   - If you're not sure, just skip it for now
+   - Can be updated later in `alokai.config.json`
+
+### What the Command Does
+
+When you create a new store, the CLI:
+
+1. **Updates Configuration Files**
+   - Creates app-specific config files (`.prettierrc.mjs`, `eslint.config.mjs`, etc.)
+   - For deployable stores:
+     - Adds store entry to `alokai.config.json`
+     - Updates `turbo.json` with build tasks and dependencies
+
+2. **Creates Store Directory Structure**
+   - Sets up store directory with all required apps
+
+#### `alokai.config.json` Updates
+
+For deployable stores, the CLI adds a new entry with default settings. All of these settings can be modified at any time:
+
+```json
+{
+  "$schema": "./.alokai/alokaiConfigSchema.json",
+  "stores": {
+    "my-brand-us": {
+      "deployment": {
+        "frontend": "nextjs",         // Which frontend to deploy
```

Comment:
```suggestion
        "frontend": "nextjs",         // Which frontend to deploy
```

not clear to me what does it mean. Is it a frontend framework? If that's the case it should be called `framework`, if its something else, specify what it is in the comment

Reply:
ok, I've created a task in the backlog https://alokai.atlassian.net/browse/TF-436